data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
The point where I was concerned about the confusion arising is if in the future we change the URLs for people on the new static ShineYourEye to some different scheme. Then we'd need to switch to getting those identifiers from a different source than the old Pombola site anyway, and changing the meaning of `source__shineyoureye` would confuse me more than deprecating one called `source__shineyoureye_pombola_slug` or `source__shineyoureye_pombola` and adding a new one called `source__shineyoureye_static` (or whatever) at that point. I can see that you might take the opposite point of view, though: that there'll only be one shineyoureye.org at a time, and so it's less confusing to just have one identifier scheme that always can be used to generate URLs to shineyoureye.org.

The internal Pombola IDs may indeed be a red herring - we don't have an immediate use for them, hence why I said "might well be useful" rather than "will be useful" :)